cart/domain/cart/paymentselection.go

```diff
@@ -21,6 +29,9 @@ type (
 		Method     string
 	}
 
+	// PaymentSplitJSON - representation of complete payment split
+	PaymentSplitJSON map[string]price.Charge
```

Comment:
Do we actually need this as a public struct? Looks more like a helper struct that is only used in the marshalling functions..

Reply:
Removed helper struct, thanks for the hint